pkg/application/registryConfigUseCase.go

```diff
@@ -133,17 +137,27 @@ func (useCase *EcosystemRegistryUseCase) applyDoguConfigDiffs(ctx context.Contex
 func (useCase *EcosystemRegistryUseCase) applySensitiveDoguConfigDiffs(ctx context.Context, doguName common.SimpleDoguName, diffs domain.SensitiveDoguConfigDiffs) error {
```

Comment:
Is this specific for a single dogu just becuase the state diff struct is grouped this way? Maybe the code gets smaller and easier to understand, if you let the repositories group by dogu.